### ISSUE TITLE
[Adaptive Cards-Schema Explorer]: state is not announced by the narrator when the button is expanding/Collapsing.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7868]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.On the Narrator using "Control +Windows +Enter" short key.
3.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
4.Navigate to 'Action.Togglevisibility" link present under Actions from the left navigation using the tab key and activate it using Enter key.
5. Navigate to the adaptive cards present under it using the Tab key.
6. Navigate to the "please confirm your data" heading under Adaptive Cards using Tab key.
7. Navigate using tab key through the buttons present under it using narrator.
8. Navigate to the "..." button and activate it using Enter key and observe the issue. 

Actual Result:
Narrator is announcing the state of the button when it got expanded/Collapsed.

Expected Result:
Narrator should give the state of the button when it got expanded/collapsed as more option button got expanded/collapsed.

User Experience:
Screen reader users will be impacted if state is not announced by the narrator as user will not be able to understand whether button got collapsed or expanded.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/190392309-76aa2f14-5ad6-4e5d-97af-fb05b21a6926.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
Added A11yTTValidated tag as issue is not repro.
